### PR TITLE
Fix transpilation

### PR DESCRIPTION
**Pull request by DenysLuko**

### CODE REVIEW

File: src/sass/general/forms-v2.scss [hunk 1]
```diff
@@ -750,7 +750,6 @@
       }
 
       &:after {
-        @extend %insetElement;
```
Comment (guyfedwards) [inline src/sass/general/forms-v2.scss]: Is this not needed at all?

Comment (DenysLuko) [inline src/sass/general/forms-v2.scss]: I didn't find the definition anywhere

**Comment by guyfedwards** [comment]
Looks good from me, think @igoratron and/or @RazvanDH should also take a look as well

**Comment by RazvanDH** [comment]
please remove all the `dist` files

**Comment by lroliphant** [comment]
Thanks @DenysLuko - looks good once `dist` is removed 👍

**Comment by DenysLuko** [comment]
@RazvanDH @lroliphant Not a sass-common expert but to me it looks a lot like bower.json is configured to pull the dist folder only which is the added to the build process in static-content. Why then remove it?

**Comment by RazvanDH** [comment]
@DenysLuko we're building and tagging on master after the branch is merged. It's easier to do the code review without the `dist`.

**Comment by DenysLuko** [comment]
@RazvanDH Fair enough, please review

**Comment by RazvanDH** [comment]
**major**
image

File: src/sass/_mixins/structure.scss [hunk 2]
```diff
@@ -304,26 +319,11 @@ $colRatio: 1;
...
-   } @else {
-     & > * {
-      @include tableCellV2($verticalAlign, $cols, $padding);
-     }
-   }
-  }
-
   @if $ie {
-    @include dtStyle;
+    @include dtStyle($row, $verticalAlign, $cols, $padding);;
```
Comment (guyfedwards) [inline src/sass/_mixins/structure.scss]: double semi-colons

File: src/sass/_mixins/structure.scss [hunk 3]
```diff
@@ -304,26 +319,11 @@ $colRatio: 1;
...
   } @else {
     @include breakPoints($minPoint, false) {
-      @include dtStyle;
+      @include dtStyle($row, $verticalAlign, $cols, $padding);;
```
Comment (guyfedwards) [inline src/sass/_mixins/structure.scss]: double semi-colons

**Comment by DenysLuko** [comment]
@RazvanDH You sure you ran npm install? Doesn't seem like it.

**Comment by RazvanDH** [comment]
**LGTM**

**Pull request merged**